code/modules/library/lib_items.dm

```diff
@@ -96,6 +96,58 @@
 		icon_state = "book-5"
 
 
+/obj/structure/bookcase/libraryspawn
+	var/spawn_category
+	var/spawn_amount = 3
+
+/obj/structure/bookcase/libraryspawn/Initialize()
+    . = ..()

```

Comment:
starting with this line, indentation is done with spaces. Please re-indent with tabs.